apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -336,12 +340,23 @@ export function createApi(uri: string, networkId: NetworkID): NetworkApi {
       return formatSpace(data.space, networkId);
     },
     loadUser: async (id: string): Promise<User | null> => {
+      const {
+        data: { follows }
+      }: { data: { follows: { space: { id: string } }[] } } = await apollo.query({
+        query: USER_FOLLOWS_QUERY,
+        variables: {
+          first: 25,
+          follower: id
+        }
+      });
+
       // NOTE: missing proposal/vote count on offchain
       return {
         id,
         proposal_count: 0,
         vote_count: 0,
-        created: 0
+        created: 0,
+        follows: follows.map(follow => follow.space.id)
```

Comment:
This for now is nested under User, to have the simplest structure.

This PR https://github.com/snapshot-labs/sx-monorepo/issues/282 will make the Follow a first class citizen, with a dedicated API to `loadFollows`, and a `follow` and `unfollow` functions from sx.js

Reply:
Follow upgrade to first class citizen, in anticipation of #282